docs/DeveloperGuide.md

```diff
@@ -267,15 +267,29 @@ Inventory managers who are proficient with typing for small F&B businesses
 
 
 ### User stories
+Add Inventory Item:
 
-Updating inventory items
+Delete Inventory Item:
+
+Update inventory items:
 1. As a purchasing manager, I can rename my inventory items, so that I can update items with an incorrect name.
 2. As a purchasing manager, I can set the inventory quantity of my items.
 3. As a purchasing manager, I can set the date I bought my inventory, so that I know how long I have been keeping certain items..
 4. As a purchasing manager, I can set the date my inventory will expire, so that I know when certain items need to be consumed or used. 
 5. As a purchasing manager, I can increase the inventory quantity of my items, so that I can keep my stock updated when I purchase new items.. 
```

Comment:
Is there a reason why some lines end with 2 full stops and some end with 1?

Reply:
Will remove